### DISCUSSION TITLE
Probleme bei der Einrichtung / Netzwerkverbindung geht verloren oder ist sehr langsam

### DISCUSSION THREAD

**Discussion by Sentinel-sys** [jomjol/AI-on-the-edge-device#2458] [Q&A] [answered]
Hallo zusammen,

ich habe letzte Woche endlich mal mit diesem genialen Projekt begonnen und habe dazu mehrere ESP32-Cam Module geflasht.
Der Flash Vorgang ging auch generell ohne Probleme nur bei der Einrichtung komme ich nicht weiter.
Entweder ist die Weboberfläche sehr langsam oder die Wifi Verbindung geht immer mal wieder verloren. Das äußert sich wie folgt:
- WebUI lädt nicht oder nur sehr langsam, Bilder werden überhaupt nicht oder nur sporadisch angezeigt
- Kamerabild bekomme ich gar keines
- Ping auf die IP des ESP geht immer mal wieder verloren und kommt auch manchmal überhaupt nicht wieder.
- Ersteinrichtung ist so unmöglich

Was habe ich schon versucht:
- ESP32-Cam mehrmals neu geflasht. Dabei habe ich das Wifi manuell eingerichtet aber auch die Methode über den AP Mode probiert
- Verschiedene ESP32-Cam Module getestet. Ich hatte noch Module zu Hause und habe festgestellt das diese scheinbar 2 MB Speicher haben. Danach habe ich mir welche mit 8 MB geholt in dem Wissen das ich eh nur 4 MB davon nutzen kann. In der seriellen Konsole bekomme ich auch entsprechend 4 MB angezeigt.
- Wifi Netzwerk getestet. Dazu habe ich die Module mit einem ESP-Cam-Webserver geflasht. Das hat ganz gut funktioniert und ich hatte nicht die oben genannten Probleme.

Aktuell weiß ich gerade nicht mehr wo ich noch ansetzen sollte. Vielleicht weiß hier jemand Rat?
Vielen Dank!

**Comment by d3nn3s08** [comment]
wie weit ist dein nächster wifi point weg ? WebUI lädt bei mir auch sehr langsam :/ bei der erst Einrichtung musste ich 10 min warten bis ich was machen konnte meiner leuft jetzt seit fast 20 tage durch und in HA mqtt daten kommen an aber mach mal auf fehlerhaft

**Comment by caco3** [comment]
Wie gut ist dein RSSI? 
Und hast Du eine externe Antenne?
Das RSSI sollte idealerweise besser als -75 dBm sein.

**Comment by Sentinel-sys** [comment, marked as answer]
Hi zusammen,

@d3nn3s08 
Ich hatte es sogar direkt neben einem Access Point probiert, leider ohne Erfolg. Mittlerweile habe ich aber glaube ich das Problem gefunden.

1. Trotz mehrfacher Aussage vom Händler sind die ESP-CAM Module leider nur mit 2MB RAM ausgestattet. Mittlerweile habe ich ein 4MB Modul.
2. Die ganze Einrichtung habe ich an meinem Firmennotebook versucht. Scheinbar habe ich hier aber eine ziemlich strikte Sicherheitseinstellung im Browser. Ich weiß nicht genau was es war was blockiert worden ist. Allerdings habe ich mir zum Schluss einfach eine VM installiert und die Einrichtung dann dort beendet. Das ging ohne Probleme.

@caco3 
Mein RSSI ist bei -73 dBm und wie gesagt habe ich aktuell keine Probleme. Externe Antenne habe ich leider nicht. Ich bin zu grobmotorisch um diesen Mini Null Ohm Wiederstand umzulöten...

Vielen Dank noch einmal an alle.

PS: Pro Tipp. Lasst das "Round Interval" auf 5 Minuten stehen. Ich hatte es auf 1 Minute gestellt und der ESP ist nach ein paar Stunden abgestürzt.
